docs/api_usage.md

```diff
@@ -44,3 +44,77 @@ Accepts following parameters:
   "status": "ok"
 }
 ```
+
+```http
+POST /api/v1/planning/plan/trigger
+```
+
+Accepts following payload:
+
+Type: application/json
+
+| Parameter | Type | Description |
+| --------- | ---- | ------------|
+| release_name          | string     | Release name to trigger plans in (can be mixed vith version to narrow filter)            |
+| plan_id          | string     | Specific plan id to trigger            |
+| version          | string     | Target scylla version of plans to trigger             |
+| common_params          | string     | Common parameters, such as backend            |
+| params          | string     | specific job parameters            |
+
+Example payload:
+
+```json
+{
+  {
+  "version": "2024.3.1~rc0",
+  "release": "scylla-master",
+  "plan_id": "some-plan-uuid",
+  "common_params": {
+      "instance_provision": "spot",
+  },
+  "params": [
+      {
+          "test": "longevity",
+          "backend": "aws",
+          "region": "eu-west-1",
+          "scylla_ami_id": "ami-abcd"
+      },
+      {
+          "test": "longevity",
+          "backend": "azure",
+          "region": "us-east1",
+          "azure_image_id": "/subscriptions…",
+      }
+  ]
+  }
+
+}
+```
+
+```json
+{
+  "response": {
+    "jobs": [
+      "http://jenkins/path/job/one/1",
+      "http://jenkins/path/job/two/3",
+      "http://jenkins/path/job/three/5",
+      "http://jenkins/path/job/four/9"
+    ],
+    "failed_to_execute": [
+      "path/job/one",
+      "path/job/two",
+      "path/job/three",
+      "path/job/four"
+    ]
+  }, 
+  "status": "ok"
+}
+```
+
+Additionally, this endpoint is available inside `argus-client-generic` executable, as follows:
+
+```bash
+argus-client-generic trigger-jobs --api-key $key --version $version --plan_id $id --release_name $release_name --job-info-file $file_path
```

Comment:
`--release-name` is in CLI, shouldn't we switch to `release` like in api calls?

Reply:
Fixed